src/main/java/org/terracotta/statistics/jsr166e/LongAdder.java

```diff
@@ -33,7 +33,7 @@
  * collection keys.
  *
  * <p><em>jsr166e note: This class is targeted to be placed in
- * java.util.concurrent.atomic<em>
+ * java.util.concurrent.atomic</em>
```

Comment:
Yes. Me too. The problem is that Doug had errors in his javadoc. So it doesn't compile. Let me try just to exclude the package. Might be wiser.